test/source/tests/setup.ts

```diff
@@ -625,8 +625,6 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await myKeyFrame.waitAll('@content-fingerprint');
       const fromKm = MOCK_KM_LAST_INSERTED_KEY[acct];
       expect(fromKm).to.exist;
-      const k = await KeyUtil.parse(fromKm.publicKey);
-      expect(await myKeyFrame.read('@content-fingerprint')).to.equal(Str.spaced(k.id));
```

Comment:
You can bring these two back, but use `fromKm.privateKey`. Public and private keys in a key pair have the same id (fingerprint). So you can use either to test the UI.